templates/settings.py.erb

```diff
@@ -24,7 +24,7 @@ DATABASES = {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': '<%= scope['pulpcore::postgresql_db_name'] %>',
         'USER': '<%= scope['pulpcore::postgresql_db_user'] %>',
-        'PASSWORD': '<%= scope['pulpcore::postgresql_db_password'] %>',
+        'PASSWORD': <%= scope.call_function('to_python', [scope['pulpcore::postgresql_db_password']]) %>,
```

Comment:
is that the function from stdlib? Then maybe use the new namespaced version?
```suggestion
        'PASSWORD': <%= scope.call_function('stdlib::to_python', [scope['pulpcore::postgresql_db_password']]) %>,
```

Reply:
it is, the rest of the file is no using the namespaced version, so I didn't want to diverge